app/src/main/java/com/example/sqlliteapp/MainActivity.java

```diff
@@ -42,7 +42,7 @@ public void onClick(View view) {
                 if(deletedrows>0)
```

Comment:
create a local func called initViews() and shift all your view initialization(findviewbyids) inside this func,